testing/jupyterhub_config.py

```diff
@@ -10,7 +10,7 @@
 c.JupyterHub.authenticator_class = DummyAuthenticator
 
 try:
-    from simplespawner import SimpleLocalProcessSpawner
+    from jupyterhub.spawners import SimpleSpawner
 except ImportError:
```

Comment:
Almost there! This can now remove the `try:...except:` because the import can't fail anymore:

```python
from jupyterhub.spawners import SimpleSpawner
c.JupyterHub.spawner_class = SimpleSpawner
```

Reply:
@minrk I have done the suggested changes. Kindly, have a look.

Reply:
Thanks! Sorry, I think I was unclear. I mean that the whole `try:...` block could be removed because there is no longer a need for the `jupyterhub-singleuser` package.

So instead of:

```python
try:
    from jupyterhub.spawners import ...
except ImportError:
    ....
```

it should have no try or except at all:

```python
from jupyterhub.spawners import ...
c.JupyterHub....
```